src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -86,7 +94,5 @@ public RsFlash(final String msg, final Level level, final String cookie) {
             )
         )
         );
-        assert level != null;
```

Comment:
I think the parameter had not been used before. To prevent the call constructor with null level